_data/tools.yml

```diff
@@ -432,6 +432,17 @@
   v2: false
   v3: true
 
+- name: meeshkan
```

Comment:
Are we doing it like `meeshkan` = product, `Meeshkan` = company? That hasn't been clear to me so far.

Reply:
That's how I've understood it! It seems less commercial when you write in lowercase :D